pkg/scheduler/scheduler.go

```diff
@@ -468,6 +468,7 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 				if status.Code() == framework.Error {
 					klog.ErrorS(nil, "Status after running PostFilter plugins for pod", "pod", klog.KObj(pod), "status", status)
 				} else {
+					fitError.Diagnosis.PostFilterMsg = status.Message()
```

Comment:
Sorry, I'm a litter confused, this is called after `Preempt()`,  so https://github.com/kubernetes/kubernetes/blob/5c79e4ccb3b57d09b9eefeec181800820a8f5fd6/pkg/scheduler/framework/types.go#L252-L255 will always false, right?

I found the `fitError.Error()` was called here:https://github.com/kubernetes/kubernetes/blob/876d4e0ab029ac7c314bb0e033bdd036531fe426/pkg/scheduler/framework/preemption/preemption.go#L173

Reply:
These are different `fitError` in Filter and PostFilter.
PostFilter just uses the `fitError' construct and changes it to a string when err is returned.
I use the `fitError` returned in RunFilter. It will be the msg in scheduling failed event.
https://github.com/kubernetes/kubernetes/blob/876d4e0ab029ac7c314bb0e033bdd036531fe426/pkg/scheduler/scheduler.go#L315

Reply:
thanks. understood.

Reply:
thanks, understood.